src/components/theme/PreviewImage/PreviewImage.stories.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PreviewImage from './PreviewImage';
+import Wrapper from '@plone/volto/storybook';
+
+const PreviewImageComponent = (props) => {
+  const { item, size = 'preview', alt, ...rest } = props;
+  return (
+    <Wrapper location={{ pathname: '/folder2/folder21/doc212' }}>
+      <div className="ui segment form attached" style={{ width: '400px' }}>
+        <PreviewImage item={item} alt={alt} size={size} {...rest} />
+      </div>
+    </Wrapper>
+  );
+};
+
+const item = {
+  '@id': '/static/media/.storybook/static/preview.png',
```

Comment:
@danalvrz Is the nebula image you added in this PR in the public domain? If not, we should replace it with an image that we are allowed to use.